src/tests/electron/tests/automated-checks-view.test.ts

```diff
@@ -73,10 +73,16 @@ describe('AutomatedChecksView', () => {
         await assertExpandedRuleGroup(3, 'TouchSizeWcag', 1);
     });
 
-    it('should not contain any accessibility issues', async () => {
-        const violations = await scanForAccessibilityIssues(automatedChecksView);
-        expect(violations).toStrictEqual([]);
-    });
+    it.each([true, false])(
+        'should pass accessibility validation with highContrastMode=%s',
+        async highContrastMode => {
+            await app.setHighContrastMode(highContrastMode);
```

Comment:
There might be an ordering issue here, since the test is only waiting for certain views to be visible before starting, but renderer-initializer runs its the initial render before the window variable for the user config controller is getting set. Maybe we should reorder those?

Reply:
I'm hopping I got this right. I updated `renderer-initiazlier` so we instanciate and assign the `UserConfigurationController` before we call `renderer.render()`.